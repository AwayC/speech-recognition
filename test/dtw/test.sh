# bin/bash
time python3 dtw.py>../../cmake-build-debug/py.out

