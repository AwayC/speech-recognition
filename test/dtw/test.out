16934 4803 1634 1526 14350 17107 15392 11547 14308 12270 1267 5821 19808 8346 16350 1569 21108 26898 11880 5168 3189 24030 13700 6086 8698 21120 20884 5923 12877 3460 232 3080 
45376 11874 7332 12047 18927 21608 27623 25587 32829 16155 31202 22329 44243 29378 21272 23303 29689 32220 18633 19342 22666 39018 17165 6908 28854 43318 38316 9353 39395 13179 15963 12900 
328936
